packages/block-library/src/template-part/edit/use-template-part-post.js

```diff
@@ -24,13 +24,14 @@ export default function useTemplatePartPost( postId, slug, theme ) {
 			// relevant file.
 			if ( slug && theme ) {
 				const cleanedSlug = cleanForSlug( slug );
+				const themeSlug = `pub-${ theme }`;
```

Comment:
**Note:**

The theme slug previously wasn't accounting for themes with parent directories. The current FSE block themes were placed in the `pub` directory, and as a result, their slugs looked like `pub-seedlet-blocks`. We, however, were querying for `seedlet-blocks` (without the pub prefix).

I wonder if we can pull the theme slug from the themes rest api rather than hard coding it here. I'll try and explore that as well